src/utils/string.spec.js

```diff
@@ -75,4 +83,13 @@ describe('utils/string', () => {
   "bar"
 ]`)
   })
+
+  it('suffixClass() works', async () => {
+    const base = 'foo'
+    expect(suffixClass(base, 'bar')).toBe('foo-bar')
+    expect(suffixClass(base, '')).toBe('foo-')
```

Comment:
This would should filter out the empty suffix and be just `foo`?

Reply:
I wanted this behavior to prevent that we get only the base branch back.
But nor really sure on that...

Reply:
It is probably never going to happen, as we always text for truthyness of the suffix

Reply:
So maybe it is OK as is.

Reply:
Lets leave it as is without filtering....

Reply:
Yeah. I think it's fine like that for now.